# Individual Deductions

**Selected Exercises**

McGraw Hill's Taxation of Individuals and Business Entities 2025

Spilker, Ayers, Lewis, Weaver, Barrick, Robinson, Worsham

In [1]:
import os
import sys
import pandas as pd
from IPython.display import display, HTML

mod_path = os.path.abspath(os.path.join(".."))
if mod_path not in sys.path:
    sys.path.append(mod_path)

from income_tax import EducationLoanInterestLimitation, CharitableContributions

---

##### **Clem is married and is a skilled carpenter. Clem’s wife, Wanda, works part time as a substitute elementary school teacher. Determine the amount of Clem’s expenses that are deductible for AGI this year (if any) under the following independent circumstances:**

- a) Clem is self-employed, and this year he incurred 525 in expenses for tools and supplies related to his job. Because neither was covered by a qualified health plan, Clem paid health insurance premiums of 3,600 to provide coverage for himself and Wanda (not through an exchange).
    - **Answer:** Clem would be able to deduct the 525 business expense as a for AGI (above the line) and 3,600 of insurance premiums as an above the line deduction, for a total of 4,125 For AGI deduction for the year.


- b) Clem and Wanda own a garage downtown that they rent to a local business for storage. This year they incurred expenses of 1,250 in utilities and 780 in depreciation.
    - **Answer:** Clem and Wanda would be able to deduct 1,250 utilities expense and 780 depreciation expense for a total of 2,030 for AGI deduction.


- c) Clem paid self-employment tax of 15,300 (the employer portion is 7,650), and Wanda had 3,000 of Social Security taxes withheld from her pay.
    - **Answer:** Clem would be able to deduct the employer portion of the self-employment tax (7,650) as a For AGI deduction.  Wanda's 3,000 are not deductible.


- d) Clem paid 45 to rent a safety deposit box to store his coin collection. Clem has collected coins intermittently since he was a child, and he expects to sell his collection when he retires.
    - **Answer:** Clem may not deduct the payment of 45 to rent a safety deposit related to his hobby expenses.

---

##### **Don Juan, a single taxpayer, is the sole owner of DJ’s Inc., an S corporation. In 2024, DJ’s Inc. incurred a massive 600,000 business loss, all of which is allocable to Don Juan as the sole shareholder. Assume that the 600,000 loss is not limited by the basis, at-risk, or passive loss rules and that Don Juan has no other business income or business losses. How much of the 600,000 loss will Don Juan be able to deduct this year? What happens to any loss not deducted this year?**

In [2]:
business_loss = 600000
business_loss_limit = 305000 # Single taxpayers 2024 threshold
deductible = min(business_loss, business_loss_limit)
excess = business_loss - deductible

print(f"""Assuming that the $600,000 loss is not limited by basis, at-risk or 
passive loss rules and that Don Juan has no other business income or business
losses:\n""")

print(f"{"Business loss":<26} | {business_loss:>8,.0f}")
print(f"{"Business loss limitation":<26} | {business_loss_limit:>8,.0f}")
print(f"{"-"*26:<26} | {"-"*8:>8}")
print(f"{"Deductible business loss":<26} | {deductible:>8,.0f}")

print(f"\nDon Juan may carry forward to subsequent years the {excess:,.0f} excess business loss.")

Assuming that the $600,000 loss is not limited by basis, at-risk or 
passive loss rules and that Don Juan has no other business income or business
losses:

Business loss              |  600,000
Business loss limitation   |  305,000
-------------------------- | --------
Deductible business loss   |  305,000

Don Juan may carry forward to subsequent years the 295,000 excess business loss.


---

##### **Smithers is a self-employed individual who earns 30,000 per year in self-employment income. Smithers pays 2,200 in annual health insurance premiums (not through an exchange) for his own medical care. In each of the following situations, determine the amount of the deductible health insurance premium for Smithers before any AGI floor limitation.**

- **a) Smithers is single and the self-employment income is his only source of income.**
    - **Answer:** Smithers may claim the 2,200 in annual health insurance premiums as a For AGI deduction.

- **b) Smithers is single, but besides being self-employed, Smithers is also employed part time by SF Power Corporation. This year Smithers elected not to participate in SF’s health plan.**
    - **Answer:** In this case, Smithers would not be able to claim health insurance premiums as a For AGI deduction because he was eligible to participate in his employer's health insurance plan but decided not to.

- **c) Smithers is self-employed and is also married. Smithers’s spouse, Samantha, is employed full time by SF Power Corporation and is covered by SF’s health plan. Smithers is not eligible to participate in SF’s health plan.**
    - **Answer:** Smithers may claim the 2,200 in annual health insurance premiums as a For AGI deduction, because he is not eligible to participate in his wife's employer provided health insurance plan.

- **d) Smithers is self-employed and is also married. Smithers’s spouse, Samantha, is employed full time by SF Power Corporation and is covered by SF’s health plan. Smithers elected not to participate in SF’s health plan.**
    - **Answer:** In this case, Smithers would not be able to claim health insurance premiums as a For AGI deduction because he was eligible to participate in his wife's employer provided health insurance plan but decided not to.

---

##### **Hardaway earned 100,000 of compensation this year. He also paid (or had paid for him) 3,000 of health insurance (not through an exchange). What is Hardaway’s AGI in each of the following situations? (Ignore the effects of Social Security and self-employment taxes.)**

- **a) Hardaway is an employee, and his employer paid Hardaway’s 3,000 of health insurance for him as a nontaxable fringe benefit. Consequently, Hardaway received 97,00 of taxable compensation and 3,000 of nontaxable compensation.**

In [3]:
agi = 97000
print(f"Adjusted Gross Income: $ {agi:,.0f}")
print(f"\nThe employer provided health insurance premiums are excluded from gross income.")

Adjusted Gross Income: $ 97,000

The employer provided health insurance premiums are excluded from gross income.


- **b) Hardaway is a self-employed taxpayer, and he paid 3,000 of health insurance himself. He is not eligible to participate in an employer-sponsored plan.**

In [4]:
income = 100000
health_insurance = 3000
agi = income - health_insurance

print(f"{"Income Earned":<30} | {income:>8,.0f}")
print(f"{"Health Insurance Premiums":<30} | {health_insurance:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"Adjusted Gross Income":<30} | {agi:>8,.0f}")
print(f"""\nBecause Hardaway is self-employed and he is not eligible to participate
in his employer's health insurance plan, he would be able to deduct {health_insurance:,.0f}
as a For agi deduction.""")

Income Earned                  |  100,000
Health Insurance Premiums      |    3,000
------------------------------ | --------
Adjusted Gross Income          |   97,000

Because Hardaway is self-employed and he is not eligible to participate
in his employer's health insurance plan, he would be able to deduct 3,000
as a For agi deduction.


---

##### **Rocco operates a beauty salon as a sole proprietorship. Rocco also owns and rents an apartment building. In 2025, Rocco had the following income and expenses. Determine Rocco’s AGI and complete page 1 (through line 11) and Schedule 1 of Form 1040 for Rocco. You may assume that Rocco will owe 2,502 in self-employment tax on his salon income, with 1,251 representing the employer portion of the self-employment tax. You may also assume that his divorce from Svetlana was finalized in 2016.**

| Description |  Amount |
| --- | ---: |
| Interest income | $11,255 |
| Salon sales and revenue | 86,360 |
| Salaries paid to beauticians | 45,250 |
| Beauty salon supplies | 23,400 |
| Alimony paid to his ex-wife, Svetlana | 6,000 |
| Rental revenue from apartment building | 31,200 |
| Depreciation on apartment building | 12,900 |
| Real estate taxes paid on apartment building | 11,100 |
| Real estate taxes paid on personal residence | 6,241 |
| Contributions to charity | 4,237 |


In [5]:
# Salon Business Income
salon = {
    "Salon Sales and Revenue": 86360,
    "Salaries Paid to Beauticians": -45250,
    "Beauty Salon Supplies": -23400 
}

# Rental business Income
rental = {
    "Rental revenue": 31200,
    "Depreciation": -12900,
    "Real State Taxes": -11100
}

# Income items
interest_income = 11255
business_income = sum(salon.values()) + sum(rental.values())
salon_net_income = sum(salon.values())
rental_net_income = sum(rental.values())

# Adjustments & Deductions
alimony = 6000                       # Pre 2019
self_employment_taxes = 1251
gross_income = interest_income + salon_net_income + rental_net_income
adjustments = alimony + self_employment_taxes
adjusted_gross_income = gross_income - adjustments

print("Income")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"Interest Income":<30} | {interest_income:>8,.0f}")
print(f"{"Salon's Net Income":<30} | {salon_net_income:>8,.0f}")
print(f"{"Rental Building Net Income":<30} | {rental_net_income:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"Gross Income":<30} | {gross_income:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print("\nFor AGI Deductions")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"Pre-2019 Alimony":<30} | {alimony:>8,.0f}")
print(f"{"One-Half Self Employment Taxes":<30} | {self_employment_taxes:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"For AGI Deductions":<30} | {adjustments:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")
print(f"{"Adjusted Gross Income":<30} | {adjusted_gross_income:>8,.0f}")
print(f"{"-"*30:<30} | {"-"*8:>8}")


Income
------------------------------ | --------
Interest Income                |   11,255
Salon's Net Income             |   17,710
Rental Building Net Income     |    7,200
------------------------------ | --------
Gross Income                   |   36,165
------------------------------ | --------

For AGI Deductions
------------------------------ | --------
Pre-2019 Alimony               |    6,000
One-Half Self Employment Taxes |    1,251
------------------------------ | --------
For AGI Deductions             |    7,251
------------------------------ | --------
Adjusted Gross Income          |   28,914
------------------------------ | --------



- **Tax Return:** [Rocco's Tax Returns](Tax_Returns/Rocco_Tax_Return.pdf)

---

##### **Jada (age 53) and Elijah (age 60) are married, and both are self-employed. In 2024, they participate in a health insurance plan with a 3,000 annual deductible and out-of-pocket maximum of 9,000. The only other health plan they have is a vision insurance plan. Are they eligible to contribute to a health savings account? Why or why not? If so, what are their maximum HSA contribution and deduction?**

- **Answer:** No, they are not eligible to contribute to a health savings account, let's see why:

| Description | Amount | 2024 Limit | 
| --- | :---: | :---: |
|Annual Deductible | 3,000 | 3,200 | 
| Out Of Pocket Expenses | 9,000 | 16,100 |

As we can see, their annual deductible was 3,000 but the minimum deductible for 2024 was 3,200.  If their annual deductible was above the minimum, which is not, they would have been able to deduct 8,300 plus an additional 1,000 (because Elijah is older than 55).

---

##### **This year Diane intends to file a married filing jointly return. Diane received 197,500 of salary and paid 5,000 of interest on loans used to pay qualified tuition costs for her dependent daughter, Deb. This year Diane has also paid moving expenses of 4,300 and 28,300 of alimony to her ex-spouse, Jack, who she divorced in 2013.**


- **a) What is Diane’s adjusted gross income?**

In [6]:
salary = 197500
loan_interest = 5000
alimony = 28300
magi = salary - alimony
interest_limitation = EducationLoanInterestLimitation("MFS", magi, loan_interest).married_filing_jointly()
adjustments = alimony + interest_limitation
adjusted_gross_income = salary - adjustments

print("Income")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Salary":<35} | {salary:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print("\nFor AGI Deductions")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Pre-2019 Alimony":<35} | {alimony:>8,.0f}")
print(f"{"Qualified Education Loan Interest":<35} | {interest_limitation:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"For AGI Deductions Total":<35} | {adjustments:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Adjusted Gross Income":<35} | {adjusted_gross_income:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")

Income
----------------------------------- | --------
Salary                              |  197,500
----------------------------------- | --------

For AGI Deductions
----------------------------------- | --------
Pre-2019 Alimony                    |   28,300
Qualified Education Loan Interest   |    2,150
----------------------------------- | --------
For AGI Deductions Total            |   30,450
----------------------------------- | --------
Adjusted Gross Income               |  167,050
----------------------------------- | --------


- **b) Suppose that Diane also reported income of 8,800 from a half share of profits from a partnership. Disregard any potential self-employment taxes on this income. What AGI would Diane report under these circumstances?**

In [7]:
partnership_income = 8800
gross_income = salary + partnership_income
magi = gross_income - alimony
interest_limitation = EducationLoanInterestLimitation("MFS", magi, loan_interest).married_filing_jointly()
adjustments = alimony + interest_limitation
adjusted_gross_income = gross_income - adjustments

print("Income")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Salary":<35} | {salary:>8,.0f}")
print(f"{"Partnership Income":<35} | {partnership_income:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Gross Income":<35} | {gross_income:>8,.0f}")
print("\nFor AGI Deductions")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Pre-2019 Alimony":<35} | {alimony:>8,.0f}")
print(f"{"Qualified Education Loan Interest":<35} | {interest_limitation:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"For AGI Deductions Total":<35} | {adjustments:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")
print(f"{"Adjusted Gross Income":<35} | {adjusted_gross_income:>8,.0f}")
print(f"{"-"*35:<35} | {"-"*8:>8}")

Income
----------------------------------- | --------
Salary                              |  197,500
Partnership Income                  |    8,800
----------------------------------- | --------
Gross Income                        |  206,300

For AGI Deductions
----------------------------------- | --------
Pre-2019 Alimony                    |   28,300
Qualified Education Loan Interest   |    1,417
----------------------------------- | --------
For AGI Deductions Total            |   29,717
----------------------------------- | --------
Adjusted Gross Income               |  176,583
----------------------------------- | --------


---

##### **Lionel is an unmarried law student at State University Law School, a qualified educational institution. This year Lionel borrowed 24,000 from County Bank and paid interest of 1,440. Lionel used the loan proceeds to pay his law school tuition. Calculate the amounts Lionel can deduct for interest on higher-education loans under the following circumstances:**

In [8]:
status = "SNG"
interest = 1440

- **a) Lionel’s AGI before deducting interest on higher-education loans is 50,000.**

In [9]:
agi = 50000
limit = EducationLoanInterestLimitation(status, agi, interest).limitation()
print(f"If AGI = {agi:,.0f} the allowable interest deduction is: $ {limit:,.0f}")

If AGI = 50,000 the allowable interest deduction is: $ 1,440


- **b) Lionel’s AGI before deducting interest on higher-education loans is 89,000.**

In [10]:
agi = 89000
limit = EducationLoanInterestLimitation(status, agi, interest).limitation()
print(f"If AGI = {agi:,.0f} the allowable interest deduction is: $ {limit:,.0f}")

If AGI = 89,000 the allowable interest deduction is: $ 576


- **c) Lionel’s AGI before deducting interest on higher-education loans is 100,000.**

In [11]:
agi = 100000
limit = EducationLoanInterestLimitation(status, agi, interest).limitation()
print(f"If AGI = {agi:,.0f} the allowable interest deduction is: $ {limit:,.0f}")

If AGI = 100,000 the allowable interest deduction is: $ 0


---

##### **In each of the following independent cases, indicate the amount (1) deductible for AGI, (2) deductible from AGI, and (3) deductible neither for nor from AGI before considering income limitations or the standard deduction.**

- a) Ted paid 50 rent on a safety deposit box at the bank. In this box he kept the few shares of stock that he owned.
    - **Answer:** This is not deductible neither for nor from AGI

- b) Tyler paid 85 for minor repairs to the fence at a rental house he owned.
    - **Answer:** Deductible for AGI, rental expenses are allowed to be taken against rental income in schedule E.

- c) Timmy paid 545 for health insurance premiums this year (not through an exchange and not with pretax dollars). Timmy is employed full time and his employer paid the remaining premiums as a qualified fringe benefit.
    - **Answer:** Deductible from AGI as an itemized deduction in Schedule A.

- d) Tess paid 1,150 of state income taxes on her consulting income.
    - **Answer:** Deductible from AGI as an itemized deduction in Schedule A.

---

##### **In each of the following independent cases, indicate the amount (1) deductible for AGI, (2) deductible from AGI, and (3) deductible neither for nor from AGI before considering income limitations or the standard deduction.**

- a) Fran spent 90 for uniforms for use on her job. Her employer reimbursed her for 75 of this amount under an accountable plan (and did not report the reimbursement as wages).
    - **Answer:** Deductible neither for nor from AGI.

- b) Timothy, a plumber employed by ACE Plumbing, spent 65 for small tools to be used on his job, but he was not reimbursed by ACE.
    - **Answer:** Deductible neither for nor from AGI.

- c) Jake is a perfume salesperson. Because of his high pay, he receives no allowance or reimbursement from his employer for advertising expenses even though his position requires him to advertise frequently. During the year, he spent 2,200 on legitimate business advertisements.
    - **Answer:** Deductible neither for nor from AGI.

- d) Trey is a self-employed, special-duty nurse. He spent 120 for uniforms.
    - **Answer:** Deductible as a For AGI business expense deduction on Schedule C.

- e) Mary, a professor at a community college, spent 340 for journal subscriptions. The journals were helpful for her research activities, but she was not reimbursed for the expenditures.
    - **Answer:** Deductible neither for nor from AGI.

- f) Wayne lost 325 on the bets he made at the race track, but he won 57 playing slot machines.
    - **Answer:** Deductible as a From AGI itemized deduction to the extent of gambling winnings.

---

##### **Penny, a full-time biochemist, loves stock car racing. To feed her passion, she bought a used dirt-track car and has started entering some local dirt-track races. The prize money is pretty small (1,000 for the winner), but she really is not in it for the money. Penny reported the following income and expenses from her nights at the track:**

| Description | Amount |
| --- | :---: |
| Prize money | 2,500 |
| Expenses: | |
| Trasportation from her home to the races | 1,000 |
| Depreciation on the dirt-track car | 4,000 |
| Entry fees | 3,500 |
| Oil, gas, supplies, repairs for the dirt-track car | 2,050 |

What are the tax effects of Penny’s racing income and expenses assuming that the racing activity is a hobby for Penny?

- **Answer:** Penny would be require to report the price money of 2,500 as income, however, her hobby expenses are not deductible for income tax purposes.

---

##### **Simpson, age 45, is a single individual who is employed full time by Duff Corporation. This year Simpson reports AGI of 50,000 and has incurred the following medical expenses:**

| Description | Amount |
| --- | :---: |
| Dentist charges | 900 |
| Physician charges | 1,800 |
| Optical charges | 500 |
| Cost of eyeglasses | 300 |
| Hospital charges | 2,100 |
| Prescription drugs | 250 |
| Over-the-counter drugs (not insulin) | 450 |
| Medical insurance premiums (not through an exchange) | 775 |

- **a) Calculate the amount of medical expenses that will be included with Simpson’s itemized deductions after any applicable limitations.**

In [19]:
agi = 50000

medical_expenses = {
    "Dentist charges": 900,
    "Physician charges": 1800,
    "Optical charges": 500,
    "Cost of eyeglasses": 300,
    "Hospital charges": 2100,
    "Prescription drugs": 250,
    "Medical Insurance Premiums": 775
}

medical_expenses = sum(medical_expenses.values())
floor_limitation = agi * .075
itemized_deduction = medical_expenses - floor_limitation
print(f"Simpson will include {itemized_deduction:,.0f} of itemized deductions.\n")

print(f"{"Qualified Medical Expenses":<26} | {medical_expenses:>8,.0f}")
print(f"{"Floor limitation":<26} | {floor_limitation:>8,.0f}")
print(f"{"-"*25:<26} | {"-"*8:>8}")
print(f"{"Itemized Deduction":<26} | {itemized_deduction:>8,.0f}")

Simpson will include 2,875 of itemized deductions.

Qualified Medical Expenses |    6,625
Floor limitation           |    3,750
-------------------------  | --------
Itemized Deduction         |    2,875


- **b) Suppose that Simpson was reimbursed for 250 of the physician’s charges and 1,200 for the hospital costs. Calculate the amount of medical expenses that will be included with Simpson’s itemized deductions after any applicable limitations.**

In [20]:
reimbursements = 250 + 1200
net_medical_expenses = medical_expenses - reimbursements
itemized_deduction = net_medical_expenses - floor_limitation

print(f"Simpson will include {itemized_deduction:,.0f} of itemized deductions.\n")

print(f"{"Qualified Medical Expenses":<32} | {medical_expenses:>8,.0f}")
print(f"{"Less: Reimbursements":<32} | {reimbursements:>8,.0f}")
print(f"{"-"*32:<32} | {"-"*8:>8}")
print(f"{"Net Qualified Medical Expenses":<32} | {net_medical_expenses:>8,.0f}")
print(f"{"Floor limitation":<32} | {floor_limitation:>8,.0f}")
print(f"{"-"*32:<32} | {"-"*8:>8}")
print(f"{"Itemized Deduction":<32} | {itemized_deduction:>8,.0f}")


Simpson will include 1,425 of itemized deductions.

Qualified Medical Expenses       |    6,625
Less: Reimbursements             |    1,450
-------------------------------- | --------
Net Qualified Medical Expenses   |    5,175
Floor limitation                 |    3,750
-------------------------------- | --------
Itemized Deduction               |    1,425


---

##### **Tim is 45 years old and is considering enrolling in an insurance program that provides for long-term care insurance. He is curious about whether the insurance premiums are deductible as a medical expense. If so, he wants to know the maximum amount that can be deducted in any year.**

- **Answer:** Yes, long-term care insurance premiums are deductible as an itemized medical expense on Schedule A, but only up to an age-based annual limit.  For 2024, at age 45, that limit is 880.00.  Any premium above that isn't deductible.  The eligible amount is then subject to the usual 7.5% of AGI floor.

---

##### **Doctor Bones prescribed physical therapy in a pool to treat Jack’s broken back. In response to this advice (and for no other reason), Jack built a swimming pool in his backyard and strictly limited use of the pool to physical therapy. Jack paid 25,000 to build the pool, but he wondered if this amount could be deducted as a medical expense. Determine if a capital expenditure such as the cost of a swimming pool qualifies for the medical expense deduction.**

- **Answer:** Yes, a capital expenditure like a pool can qualify as a medical expense, but only to the extent its cost exceeds any resulting increates in the home's fair market value.  So Jack's deductible amount is 25,000 minus the pool's contribution to his home's FMV, subject to the 7.5% of AGI floor.

---

##### **Charles has AGI of 50,000 and has made the following payments related to (1) land he inherited from his deceased aunt and (2) a personal vacation taken last year. Calculate the amount of taxes Charles may include in his itemized deductions for the year under the following circumstances:**

| Description | Amount |
| --- | :---: |
| State inheritance tax on the land | 1,200 |
| Country real estate tax on the land | 1,500 |
| School disctrict tax on the land | 690 |
| City special assessment on the land (new curbs and gutters) | 700 |
| State tax on airline tickets (paid on vacation) | 125 |
| Local hotel tax (paid during vacation) | 195 |


In [22]:
real_state_tax = 1500
school_district_tax = 690

deductible_taxes = real_state_tax + school_district_tax
print(f"Deductible land taxes: $ {deductible_taxes:,.0f}")

Deductible land taxes: $ 2,190


- **a) Suppose that Charles holds the land for appreciation.**
    - **Answer:** Charles would be able to deduct 2,190 as an itemized deduction on Schedule A.

- **b) Suppose that Charles holds the land for rent.**
    - **Answer:** Charles would be able to deduct the same 2,190 but now as a For AGI deduction on Schedule E.

- **c) Suppose that Charles holds the land for appreciation and that the vacation was actually a business trip.**
    - **Answer:** The land is still held for appreciation (not rent), so the 2,190 stays as an itemized deduction on Schedule A.  Because the vacations are now a business trip, the 320.00 (125 + 195) becomes deductible as ordinary business travel expense - for AGI on Schedule C if Charles is self-employed.

---

##### **Dan has AGI of 50,000 and paid the following taxes during this tax year. Calculate how much Dan can deduct for taxes as an itemized deduction this year.**

| Description | Amount |
| --- | :---: |
| State income tax withholding | 1,400 |
| State income tax estimated payments | 750 |
| Federal income tax withholding | 3,000 |
| Social Security tax withheld from wages | 2,100 |
| State excise tax on liquor | 400 |
| Automobile license (based on the car's weight) | 300 |
| State sales tax paid | 475 |

In [25]:
state_income_tax_withholding = 1400
state_income_tax_estimated_payments = 750
local_taxes = state_income_tax_withholding + state_income_tax_estimated_payments
state_excise_tax_on_liquour = 400
state_sales_taxes = 475


limit = 10000
itemized_deduction = min(limit, max(local_taxes, state_sales_taxes))
print(f"Dan would be able to deduct ${itemized_deduction:,.0f} as an itemized deduction.")

print("\nState and Local Taxes")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"State income tax withholding":<40} | {state_income_tax_withholding:>8,.0f}")
print(f"{"State income tax estimated payments":<40} | {state_income_tax_estimated_payments:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Total Local Taxes":<40} | {local_taxes:>8,.0f}")

print("\nState Sales Taxes")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"State sales taxes paid":<40} | {state_sales_taxes:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")

print(f"\n{"Itemized deduction:":<40} | {itemized_deduction:>8,.0f}")


Dan would be able to deduct $2,150 as an itemized deduction.

State and Local Taxes
---------------------------------------- | --------
State income tax withholding             |    1,400
State income tax estimated payments      |      750
---------------------------------------- | --------
Total Local Taxes                        |    2,150

State Sales Taxes
---------------------------------------- | --------
State sales taxes paid                   |      475
---------------------------------------- | --------

Itemized deduction:                      |    2,150


---

##### **Tim is a single, cash-method taxpayer with an AGI of 50,000. In April of this year, Tim paid 1,020 with his state income tax return for the previous year. During the year, Tim had 5,400 of state income tax and 18,250 of federal income tax withheld from his salary. In addition, Tim made estimated payments of 1,360 and 1,900 for state and federal income taxes, respectively. Finally, Tim expects to receive a refund of 500 for state income taxes when he files his state tax return for this year in April next year. What is the amount of taxes that Tim can deduct as an itemized deduction?**

In [26]:
state_income_tax = 1020
state_income_tax_withholding = 5400
estimated_payments = 1360
total_taxes = state_income_tax + state_income_tax_withholding + estimated_payments
limit = 10000
itemized_deduction = min(limit, total_taxes)

print("\nState and Local Taxes")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"State income tax":<40} | {state_income_tax:>8,.0f}")
print(f"{"State income tax withholding":<40} | {state_income_tax_withholding:>8,.0f}")
print(f"{"State income tax estimated payments":<40} | {estimated_payments:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Total Local Taxes":<40} | {total_taxes:>8,.0f}")
print(f"{"Limitation":<40} | {limit:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Itemized deduction":<40} | {itemized_deduction:>8,.0f}")


State and Local Taxes
---------------------------------------- | --------
State income tax                         |    1,020
State income tax withholding             |    5,400
State income tax estimated payments      |    1,360
---------------------------------------- | --------
Total Local Taxes                        |    7,780
Limitation                               |   10,000
---------------------------------------- | --------
Itemized deduction                       |    7,780


---

##### **This year Randy paid 28,000 of interest on his residence. (Randy borrowed 450,000 to buy his residence, which is currently worth 500,000.) Randy also paid 2,500 of interest on his car loan and 4,200 of margin interest to his stockbroker (investment interest expense). How much of this interest expense can Randy deduct as an itemized deduction under the following circumstances?**

- **a) Randy received 2,200 of interest this year and no other investment income or expenses. His AGI is 75,000.**
    - **Answer:** The investment interest deduction is limited to net investment income, so Randy would be able to deduct 2,200 on interest income.  Furthermore, because 28,000 of interest paid on his loan is well below the 750,000 limit, he can fully deduct the interest paid on his mortgage. Interest on loans to acquire personal-use automobiles are not deductible.

- **b) Randy had no investment income this year, and his AGI is 75,000.**
    - **Answer:** In this case, Randy cannot deduct any investment interest expense but he can still deduct the mortgage interest.

---

##### **Janyce, a single taxpayer, has AGI of $125,000 and paid the following taxes this year. Calculate how much Janyce can deduct for taxes as an itemized deduction this year.**

| Description | Amount |
| --- | ---: |
| State income tax withholding | 7,200 |
| State income tax estimated payments | 600 |
| State income tax refund (applied to this year’s tax) | 800 | 
| State automobile tax (based on car’s value) | 1,900 |


In [14]:
limit = 10000

taxes = {
    "Tax withholdings": 7200,
    "Estimated tax payments":  600, 
    "Income tax refund": 800,
    "Automobile Tax": 1900
}
total_taxes = sum(taxes.values())
itemized_deduction = min(limit, total_taxes)

print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Total taxes paid":<40} | {total_taxes:>8,.0f}")
print(f"{"Limit":<40} | {limit:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Itemized deduction":<40} | {itemized_deduction:>8,.0f}")



---------------------------------------- | --------
Total taxes paid                         |   10,500
Limit                                    |   10,000
---------------------------------------- | --------
Itemized deduction                       |   10,000


---

##### **This year, Major Healy paid 40,000 of interest on a mortgage on his home (he borrowed 800,000 to buy the residence in 2015; 900,000 original purchase price and value at purchase), 6,000 of interest on a 120,000 home equity loan on his home (loan proceeds were used to buy antique cars), and 10,000 of interest on a mortgage on his vacation home (borrowed 200,000 to purchase the home in 2010; home purchased for 500,000). Major Healy’s AGI is 220,000. How much interest expense can Major Healy deduct as an itemized deduction?**

In [33]:
principal_home_loan = 800000
vacational_home_loan = 200000
total_loans = principal_home_loan + vacational_home_loan
principal_home_interest = 40000
vacational_home_interest = 10000
qualified_interest = principal_home_interest + vacational_home_interest

print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Principal Residence Mortgage Interest":<40} | {principal_home_interest:>8,.0f}")
print(f"{"Vactional Residence Mortgage Interest":<40} | {vacational_home_interest:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Qualified Interest":<40} | {qualified_interest:>8,.0f}")

print(f"""\nMajor Healy would be able to deduct {qualified_interest:,.0f} as an itemized deduction 
on Schedule A.  The combined acquisition debt is: {total_loans:,.0f}, which is within
the grandfathered limit (pre December 16, 2017).

The 6,000 interest paid related to the home equity loan used to buy antique cars 
are not deductible.""")

---------------------------------------- | --------
Principal Residence Mortgage Interest    |   40,000
Vactional Residence Mortgage Interest    |   10,000
---------------------------------------- | --------
Qualified Interest                       |   50,000

Major Healy would be able to deduct 50,000 as an itemized deduction 
on Schedule A.  The combined acquisition debt is: 1,000,000, which is within
the grandfathered limit (pre December 16, 2017).

The 6,000 interest paid related to the home equity loan used to buy antique cars 
are not deductible.


---

##### **Jack, who files married filing separately, has AGI of 45,000 and paid the following taxes this year. Calculate how much Jack can deduct for taxes as an itemized deduction this year.**

| Description | Amount |
| --- | ---: |
| State income tax withholding | 6,200 |
| State income tax estimated payments | 500 |
| State income tax refund (applied to this year’s tax) | 1,000 |
| State automobile tax (based on car’s weight) | 800 |

In [1]:
taxes = {
    "State Income Tax Withholdings": 6200,
    "State income tax estimated payments": 500,
    "State income tax refund": 1000,
}

limit = 5000
total_taxes = sum(taxes.values())
itemized_deduction = min(limit, total_taxes)

print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Total taxes paid":<40} | {total_taxes:>8,.0f}")
print(f"{"Limit":<40} | {limit:>8,.0f}")
print(f"{"-"*40:<40} | {"-"*8:>8}")
print(f"{"Itemized deduction":<40} | {itemized_deduction:>8,.0f}")

---------------------------------------- | --------
Total taxes paid                         |    7,700
Limit                                    |    5,000
---------------------------------------- | --------
Itemized deduction                       |    5,000


---

 ##### **Ray Ray made the following contributions in 2024.**

 | Charity | Property | Cost | FMV |
 | --- | --- | ---: | ---: |
 | Athens Academy School | Cash | 5,000 | 5,000 |
 | United Way | Cash | 4,000 | 4,000 |
 | American Heart Association | Antique Painting | 15,000 | 75,000 |
 | Religious Organization (Congregation) | Coca-Cola stock | 12,000 | 20,000 |

Determine the maximum amount of charitable deduction for each of these contributions ignoring the AGI ceiling on charitable contributions and assuming that the American Heart Association plans to sell the antique painting to fund its operations. Ray Ray has owned the painting and Coca-Cola stock since 1990.

In [6]:
contributions = {
    "Athens Academy - Cash": 5000,
    "United Way - Cash": 4000,
    "American Heart Association - Painting": 15000,
    "Congregation - Stock": 20000,
}

contributions = sum(contributions.values())

print(f"Ray may take $ {contributions:,.0f} as itemized deductions. (Ignoring AGI ceiling)")
print(f"""\n
Note: 
    - Becase the American Heart Association plans to sell the painting,  Ray must deduct his 
    basis rather than the fair market value.""")


Ray may take $ 44,000 as itemized deductions. (Ignoring AGI ceiling)


Note: 
    - Becase the American Heart Association plans to sell the painting,  Ray must deduct his 
    basis rather than the fair market value.


---

##### **Juanita paid 50,000 of interest on a mortgage on her home (loan of 1,000,000 at 5% interest rate to buy the residence in 2018; 1,200,000 original purchase price and value at purchase) and 6,500 of interest on a 100,000 home-equity loan on her home (loan proceeds were used to buy furniture). Juanita’s AGI is 600,000. How much interest expense can Juanita deduct as an itemized deduction?**

- **Answer:** 

In [8]:
debt_limit = 750000
mortgage = 1000000
interest_paid = 50000
allowable_interest = interest_paid * (750000 / 1000000)

print(f"""
Juanita would be able to deduct up to ${allowable_interest:,.0f} paid related the mortgage on her home.  
Her loan principal is $1,000,000, which exceeds the $750,000 cap. So only the interest 
allocable to the first $750,000 of debt is deductible.

The $6,500 paid on the home-equity loan is not deductible because the proceeds were used 
to buy furniture.""")



Juanita would be able to deduct up to $37,500 paid related the mortgage on her home.  
Her loan principal is $1,000,000, which exceeds the $750,000 cap. So only the interest 
allocable to the first $750,000 of debt is deductible.

The $6,500 paid on the home-equity loan is not deductible because the proceeds were used 
to buy furniture.


---

##### **Calvin reviewed his canceled checks and receipts this year (2024) for charitable contributions, which included an antique painting and IBM stock. He has owned the IBM stock and the painting since 2005.**

| Donee | Item | Cost | FMV |
| --- | --- | ---: | ---: |
| Hobbs Medical Center | IBM stock | 5,000 | 22,000 |
| State Museum | Antique painting | 5,000 | 3,000 |
| A needy family | Food and clothes | 400 | 250 |
| United Way | Cash | 8,000 | 8,000 |

Calculate Calvin’s charitable contribution deduction and carryover (if any) under the following circumstances.

- **a) Calvin’s AGI is 100,000.**

In [11]:
agi = 100000
remaining_agi = agi

cash = 8000
ordinary_property = 3000
cap_gain_property = 22000

cash_limit = agi * .60
ordinary_limit = ordinary_property * .50
cap_gain_limit = agi * .30

cash_deduction = min(cash, cash_limit, remaining_agi)
cash_carryforward = cash - cash_deduction
remaining_agi -= cash_deduction

ordinary_deduction = min(ordinary_property, ordinary_limit, remaining_agi)
ordinary_carryforward = ordinary_property - ordinary_deduction
remaining_agi -= ordinary_deduction

cap_gain_deduction = min(cap_gain_property, cap_gain_limit, remaining_agi)
cap_gain_carryforward = cap_gain_property - cap_gain_deduction
remaining_agi -= cap_gain_deduction

total_deductions = cash_deduction + ordinary_deduction + cap_gain_deduction
total_carryforward = cash_carryforward + ordinary_carryforward + cap_gain_carryforward

results = {
    "Property": ["Cash contributions", "Ordinary contributions", "Capital gain contributions"],
    "Deduction": [cash_deduction, ordinary_deduction, cap_gain_deduction],
    "Carryforward": [cash_carryforward, ordinary_carryforward, cap_gain_carryforward],
}

df = pd.DataFrame(results)
total = pd.DataFrame(([{"Property": "Total", "Deduction": total_deductions, "Carryforward": total_carryforward}]))
df = pd.concat([df, total], ignore_index=True)

df["Deduction"] = df["Deduction"].map("{:,.0f}".format)
df["Carryforward"] = df["Carryforward"].map("{:,.0f}".format)

display(HTML(df.to_html(index=False)))

Property,Deduction,Carryforward
Cash contributions,"8,000",0
Ordinary contributions,"1,500","1,500"
Capital gain contributions,"22,000",0
Total,"31,500","1,500"


- **b) Calvin’s AGI is 100,000, but the State Museum told Calvin that it plans to sell the painting.**
    - **Answer: Same as above.**

- **c) Calvin’s AGI is 50,000.**

In [16]:
agi = 50000
remaining_agi = agi

cash = 8000
ordinary_property = 3000
cap_gain_property = 22000

cash_limit = agi * .60
ordinary_limit = agi * .50
cap_gain_limit = min(agi * .30, (agi * .50) - cash_deduction - ordinary_deduction)

cash_deduction = min(cash, cash_limit, remaining_agi)
cash_carryforward = cash - cash_deduction
remaining_agi -= cash_deduction

ordinary_deduction = min(ordinary_property, ordinary_limit, remaining_agi)
ordinary_carryforward = ordinary_property - ordinary_deduction
remaining_agi -= ordinary_deduction

cap_gain_deduction = min(cap_gain_property, cap_gain_limit, remaining_agi)
cap_gain_carryforward = cap_gain_property - cap_gain_deduction
remaining_agi -= cap_gain_deduction

total_deductions = cash_deduction + ordinary_deduction + cap_gain_deduction
total_carryforward = cash_carryforward + ordinary_carryforward + cap_gain_carryforward

results = {
    "Property": ["Cash contributions", "Ordinary contributions", "Capital gain contributions"],
    "Deduction": [cash_deduction, ordinary_deduction, cap_gain_deduction],
    "Carryforward": [cash_carryforward, ordinary_carryforward, cap_gain_carryforward],
}

df = pd.DataFrame(results)
total = pd.DataFrame(([{"Property": "Total", "Deduction": total_deductions, "Carryforward": total_carryforward}]))
df = pd.concat([df, total], ignore_index=True)

df["Deduction"] = df["Deduction"].map("{:,.0f}".format)
df["Carryforward"] = df["Carryforward"].map("{:,.0f}".format)

display(HTML(df.to_html(index=False)))

Property,Deduction,Carryforward
Cash contributions,"8,000",0
Ordinary contributions,"3,000",0
Capital gain contributions,"14,000","8,000"
Total,"25,000","8,000"


- **d) Calvin’s AGI is 100,000, and Hobbs is a private nonoperating foundation.**

In [21]:
agi = 100000
remaining_agi = agi

cash = 8000
ordinary_property = 3000
cap_gain_property = 22000

cash_limit = agi * .60
ordinary_limit = agi * .50
cap_gain_limit = agi * .20

cash_deduction = min(cash, cash_limit, remaining_agi)
cash_carryforward = cash - cash_deduction
remaining_agi -= cash_deduction

ordinary_deduction = min(ordinary_property, ordinary_limit, remaining_agi)
ordinary_carryforward = ordinary_property - ordinary_deduction
remaining_agi -= ordinary_deduction

cap_gain_deduction = min(cap_gain_property, cap_gain_limit, remaining_agi)
cap_gain_carryforward = cap_gain_property - cap_gain_deduction
remaining_agi -= cap_gain_deduction

total_deductions = cash_deduction + ordinary_deduction + cap_gain_deduction
total_carryforward = cash_carryforward + ordinary_carryforward + cap_gain_carryforward

results = {
    "Property": ["Cash contributions", "Ordinary contributions", "Capital gain contributions"],
    "Deduction": [cash_deduction, ordinary_deduction, cap_gain_deduction],
    "Carryforward": [cash_carryforward, ordinary_carryforward, cap_gain_carryforward],
}

df = pd.DataFrame(results)
total = pd.DataFrame(([{"Property": "Total", "Deduction": total_deductions, "Carryforward": total_carryforward}]))
df = pd.concat([df, total], ignore_index=True)

df["Deduction"] = df["Deduction"].map("{:,.0f}".format)
df["Carryforward"] = df["Carryforward"].map("{:,.0f}".format)

display(HTML(df.to_html(index=False)))

Property,Deduction,Carryforward
Cash contributions,"8,000",0
Ordinary contributions,"3,000",0
Capital gain contributions,"20,000","2,000"
Total,"31,000","2,000"


- **e) Calvin’s AGI is 100,000, but the painting is worth 10,000.**

In [22]:
agi = 100000
remaining_agi = agi

cash = 8000
cap_gain_property = 22000 + 10000

cash_limit = agi * .60
ordinary_limit = agi * .50
cap_gain_limit = min(agi * .30, (agi * .50) - cash_deduction - ordinary_deduction)

cash_deduction = min(cash, cash_limit, remaining_agi)
cash_carryforward = cash - cash_deduction
remaining_agi -= cash_deduction

ordinary_deduction = min(ordinary_property, ordinary_limit, remaining_agi)
ordinary_carryforward = ordinary_property - ordinary_deduction
remaining_agi -= ordinary_deduction

cap_gain_deduction = min(cap_gain_property, cap_gain_limit, remaining_agi)
cap_gain_carryforward = cap_gain_property - cap_gain_deduction
remaining_agi -= cap_gain_deduction

total_deductions = cash_deduction + cap_gain_deduction
total_carryforward = cash_carryforward + cap_gain_carryforward

results = {
    "Property": ["Cash contributions", "Capital gain contributions"],
    "Deduction": [cash_deduction, cap_gain_deduction],
    "Carryforward": [cash_carryforward, cap_gain_carryforward],
}

df = pd.DataFrame(results)
total = pd.DataFrame(([{"Property": "Total", "Deduction": total_deductions, "Carryforward": total_carryforward}]))
df = pd.concat([df, total], ignore_index=True)

df["Deduction"] = df["Deduction"].map("{:,.0f}".format)
df["Carryforward"] = df["Carryforward"].map("{:,.0f}".format)

display(HTML(df.to_html(index=False)))

Property,Deduction,Carryforward
Cash contributions,"8,000",0
Capital gain contributions,"30,000","2,000"
Total,"38,000","2,000"


---

##### **In addition to cash contributions to charity, Dean decided to donate shares of stock and a portrait painted during the earlier part of the last century. Dean purchased the stock and the portrait many years ago as investments. Dean reported the following recipients in 2024:**

| Charity | Property | Cost | FMV |
| --- | --- | ---: | ---: |
| State University | Cash | $ 15,000 | $ 15,000 |
| Red Cross | Cash | 14,500 | 14,500 |
| State History Museum | Painting | 5,000 | 82,000 |
| City Medical Center | Dell stock | 28,000 | 17,000 |

##### **a) Determine the maximum amount of charitable deduction for each of these contributions ignoring the AGI ceiling on charitable contributions.**

In [2]:
# Ignoring AGI limits
deductions = {
    "State University - Cash": 15000,
    "Red Cross - Cash": 14500,
    "State History Museum": 82000,
    "City Medical Center": 17000
}

contributions = sum(deductions.values())

print(f"Dean may take $ {contributions:,.0f} as itemized deductions. (Ignoring AGI ceiling)")

Dean may take $ 128,500 as itemized deductions. (Ignoring AGI ceiling)


- **b) Assume that Dean’s AGI this year is 150,000. Determine Dean’s itemized deduction for his charitable contributions this year and any carryover.**

In [3]:
agi = 150000
remaining_agi = agi

cash = 15000 + 14500
cash_limit = agi * .60
cash_deduction = min(cash, cash_limit, remaining_agi)
cash_carryforward = cash - cash_deduction
remaining_agi -= cash_deduction

cap_gain_property = 82000 + 17000
cap_gain_limit = agi * .30
cap_gain_deduction = min(cap_gain_property, cap_gain_limit, remaining_agi)
cap_gain_carryforward = cap_gain_property - cap_gain_deduction
remaining_agi -= cap_gain_deduction

total_deductions = cash_deduction + cap_gain_deduction
total_carryforward = cash_carryforward + cap_gain_carryforward

results = {
    "Property": ["Cash contributions", "Capital gain contributions"],
    "Deduction": [cash_deduction, cap_gain_deduction],
    "Carryforward": [cash_carryforward, cap_gain_carryforward],
}

df = pd.DataFrame(results)
total = pd.DataFrame(([{"Property": "Total", "Deduction": total_deductions, "Carryforward": total_carryforward}]))
df = pd.concat([df, total], ignore_index=True)

df["Deduction"] = df["Deduction"].map("{:,.0f}".format)
df["Carryforward"] = df["Carryforward"].map("{:,.0f}".format)

display(HTML(df.to_html(index=False)))

Property,Deduction,Carryforward
Cash contributions,"29,500",0
Capital gain contributions,"45,000","54,000"
Total,"74,500","54,000"


- **c) Suppose Dean is a dealer in antique paintings and had held the painting for sale before the contribution. What is Dean’s charitable contribution deduction for the painting in this situation (ignoring AGI limitations)?**
    - **Answer:** Ignoring AGI limitations, his painting will be treated as ordinary property. His deduction would be limited to the lesser of basis or FMV. In this case he would be able to deduct his basis which is 5,000. 

- **d) Suppose that Dean’s objective with the donation to the museum was to finance expansion of the historical collection. Hence, Dean was not surprised when the museum announced the sale of the painting because of its limited historical value. What is Dean’s charitablecontribution deduction for the painting in this situation (ignoring AGI limitations)?**
    - **Answer:** Ignoring AGI limitations, and because the charity plans to sell the painting, Dean would be able to deduct the basis of the property which is 5,000.

---